sycl/include/sycl/ext/codeplay/experimental/fusion_wrapper.hpp

```diff
@@ -6,6 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+// TODO(#15184): Delete this file in the next ABI-breaking window.
```

Comment:
Same

Reply:
No, I've created an issue to track this in https://github.com/intel/llvm/issues/15184.